actions.md

```diff
@@ -0,0 +1,547 @@
+
+**[[NH: check link:]]**
+Another way of invoking an action that does not require authentication is via [web actions](webactions.md#web-actions).
+
+Any action may be exposed as a web action, using the `--web true` command line option when creating or updating an action.
+
+```
+nim action update greeting --web true
+ok: updated action greeting
+```
+
+The resource URL for a web action is different:
+```
+nim action get greeting --url
+ok: got action greeting
+https://${APIHOST}/api/v1/web/${NAMESPACE}/${PACKAGE}/greeting
+```
+
+You can use `curl` or `wget` to invoke the action.
+```
+curl `nim action get greeting --url | tail -1`.json
+{
+  "payload": "Hello, Toto from somewhere!"
+}
+```
+
+### Save action code
+
+Code associated with an existing action may be retrieved and saved locally. Saving can be performed on all actions except sequences and docker actions.
+
+1. Save action code to a filename that corresponds with an existing action name in the current working directory. A file extension that corresponds to the action kind is used. An extension of _.zip_ is used for action code that is a zip file.
+
+  **[[NH: please check command and output:]]**
+  ```
+  nim action get /whisk-system/samples/greeting --save
+  ok: saved action code to /path/to/namespace/greeting.js
+  ```
+
+2. You can provide your own file name and extension by using the `--save-as` flag.
+
+  **[[NH: check output:]]**
+  ```
+  wsk action get /whisk-system/samples/greeting --save-as hello.js
+  ok: saved action code to /path/to/namespace/hello.js
+  ```
+
+### List actions
+
+You can list all the actions that you have created using the following commnand:
+
+```
+nim action list
+  actions
+  /guest/mySequence                  private sequence
+  /guest/greeting                    private nodejs:6
+```
+
+Here, we see actions listed in order from most to least recently updated. For easier browsing, you can use the flag `--name-sort` or `-n` to sort the list alphabetically:
+
+**[[NH: Output list seems backwards for alphabetical order?]]**
+```
+nim action list --name-sort
+  actions
+  /guest/mySequence                  private sequence
+  /guest/greeting                    private nodejs:6
+```
+
+The list is now sorted alphabetically by namespace, then package name if any, and finally action name, with the default package (no specified package) listed at the top. **[[NH: Is namespace listed in this output example? What does output look like if there's a package? Does nim output list the namespace?]]**
+
+**Note**: The printed list is sorted alphabetically after it is received from the platform. Other list flags such as `--limit` and `--skip` are applied to the block of actions before they are received for sorting. To list actions in order by creation time, use the flag `--time`.
+
+To filter your list of actions to just those within a specific package, use this command:
+
+**[[NH: please check command and output:]]**
```

Comment:
For now I am not checking individual command outputs ... instead will work to get so you can run this everything for yourself.   Worst case (if we're unable to accommodate that), you leave place holders for the output and one of us will fill it in on a later pass.